gradle/libs.versions.toml

```diff
@@ -104,7 +104,8 @@ junit-platform-reporting = { module = "org.junit.platform:junit-platform-reporti
 logback-classic = { module = "ch.qos.logback:logback-classic", version.ref = "logback" }
 keycloak-admin-client = { module = "org.keycloak:keycloak-admin-client", version.ref = "keycloak" }
 maven-resolver-supplier = { module = "org.apache.maven.resolver:maven-resolver-supplier", version.ref = "mavenResolver" }
-micrometer-core = { module = "io.micrometer:micrometer-core", version = "1.13.0" }
+mariadb-java-client = { module = "org.mariadb.jdbc:mariadb-java-client", version = "3.3.3" }
+micrometer-core = { module = "io.micrometer:micrometer-core", version = "1.12.5" }
```

Comment:
Beware: you inadvertently downgraded this dependency. Please revert to 1.13.0 as before.